colcon-build-and-test/action.yml

```diff
@@ -0,0 +1,90 @@
+name: "colcon-build-and-test"
+description: "Build and test the project"
+
+inputs:
+  rosdistro:
+    description: "ROS distro"
+    required: true
+  build-depends-repos:
+    description: "VCS repositories containing build dependencies"
+    required: true
+  target-packages:
+    description: "Target packages to build and test"
+    required: true
+  github-access-token:
+    description: "If the repository is private, specify a token."
+    required: false
+outputs:
+  compile-commands-hash:
+    description: Calculated hash of compile_commands.json
+    value: ${{ steps.calc-hash.outputs.hash }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      if: ${{ inputs.target-packages != '' }}
```

Comment:
I think you can remove this check because it can be checked in the caller side.

Reply:
Removed 2479eb6